packages/clerk-js/src/core/clerk.ts

```diff
@@ -1324,6 +1312,24 @@ export default class Clerk implements ClerkInterface {
     });
   };
 
+  #buildSignInOrUpUrl = (key: 'signInUrl' | 'signUpUrl', options?: RedirectOptions): string => {
+    const opts: RedirectOptions = {
+      afterSignInUrl: extractAuthPathByPriority('afterSignInUrl', { ctx: options, options: this.#options }, false),
+      afterSignUpUrl: extractAuthPathByPriority('afterSignUpUrl', { ctx: options, options: this.#options }, false),
+      redirectUrl: options?.redirectUrl || window.location.href,
+    };
+    if (!this.#environment || !this.#environment.displayConfig) {
```

Comment:
❓ Should we replace this condition with an early guard (first check in the function) checking if Clerk.isLoaded?